pkg/metrics/metrics.go

```diff
@@ -54,6 +62,8 @@ func initMetrics() {
 	UserSignupAutoDeactivatedTotal = newCounter("user_signups_auto_deactivated_total", "Total number of Automatically Deactivated User Signups")
 	// Gauges
 	MasterUserRecordGauge = newGauge("master_user_record_current", "Current number of Master User Records")
+	// GaugeVecs
+	MasterUserRecordGaugeVec = newGaugeVec("master_user_records_current", "Current number of Master User Records", "cluster_name")
```

Comment:
it's very minor and you can ignore it, but it would be fine if the var name was similar to the name of the metric:
```suggestion
	MasterUserRecordsGaugeVec = newGaugeVec("master_user_records_current", "Current number of Master User Records", "cluster_name")
```

Reply:
actually, I renamed it to `UserAccountGaugeVec` since it is associated with the current number of UserAccount resources